Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -0,0 +1,219 @@
+        didSet {
+            if isLoaded {
+                UIView.transitionFromView(interstitialView, toView: webView,
+                    duration: 0.5,
+                    options: UIViewAnimationOptions.TransitionCrossDissolve,
+                    completion: { finished in
+                        self.interstitialView.removeFromSuperview()
+                        self.interstitialSpinnerView.stopAnimating()
+                })
+            }
+        }
+    }
+
+    private var isError: Bool = false {
+        didSet {
+            if isError {
+                interstitialErrorView.hidden = false
+                UIView.transitionFromView(interstitialSpinnerView, toView: interstitialErrorView,
+                    duration: 0.5,
+                    options: UIViewAnimationOptions.TransitionCrossDissolve,
+                    completion: { finished in
+                        self.interstitialSpinnerView.removeFromSuperview()
+                        self.interstitialSpinnerView.stopAnimating()
+                })
+            }
+        }
+    }
+
+    // The view shown while the content is loading in the background web view.
+    private var interstitialView: UIView!
+    private var interstitialSpinnerView: UIActivityIndicatorView!
+    private var interstitialErrorView: UILabel!
+
+    // The web view that displays content.
+    var webView: WKWebView!
+
+    private func startLoading(timeout: Double = DefaultTimeoutTimeInterval) {
+        if self.isLoaded {
+            return
+        }
+        if timeout > 0 {
+            self.timer = NSTimer.scheduledTimerWithTimeInterval(timeout, target: self, selector: "SELdidTimeOut", userInfo: nil, repeats: false)
+        } else {
+            self.timer = nil
+        }
+        self.webView.loadRequest(NSURLRequest(URL: url))
+        self.interstitialSpinnerView.startAnimating()
+    }
+
+    init(backgroundColor: UIColor = UIColor.whiteColor(), title: NSAttributedString? = nil) {
+        interstitialBackgroundColor = backgroundColor
+        settingsTitle = title
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Don't extend under navigation bar.
+        self.edgesForExtendedLayout = UIRectEdge.None
+
+        // This background agrees with the web page background.
+        // Keeping the background constant prevents a pop of mismatched color.
+        view.backgroundColor = interstitialBackgroundColor
+
+        self.webView = makeWebView()
+        view.addSubview(webView)
+
+        // Destructuring let causes problems.
+        let ret = makeInterstitialViews()
+        self.interstitialView = ret.0
+        self.interstitialSpinnerView = ret.1
+        self.interstitialErrorView = ret.2
+
+        view.addSubview(interstitialView)
+        if debug {
```

Comment:
Was this debug flag for testing locally? If so, do we need it as part of this feature?